builtin/rebase--interactive.c

```diff
@@ -224,6 +226,15 @@ int cmd_rebase__interactive(int argc, const char **argv, const char *prefix)
 		warning(_("--[no-]rebase-cousins has no effect without "
 			  "--rebase-merges"));
 
+	if (opts.ignore_whitespace) {
+		struct strbuf buf = STRBUF_INIT;
+		if (raw_strategies)
```

Comment:
raw_strategies is an example of a variable that has disappeared since the old version you are basing your patch on, which will cause you some pain when you try to rebase on a newer version.  In general, when working on git.git you want to be on a relatively recent version of master.  If you know of conflicting changes others have submitted in 'next' or 'pu', you may even want to base your work on those.